scwx-qt/source/scwx/qt/gl/draw/placefile_text.cpp

```diff
@@ -136,7 +136,11 @@ void PlacefileText::Impl::RenderTextDrawItem(
          std::chrono::system_clock::now() :
          selectedTime_;
 
-   if ((!thresholded_ || mapDistance_ <= di->threshold_) &&
+   const bool thresholdMet =
```

Comment:
Consider adding `!thresholded_` to the front of this check, to optimize and skip performing evaluation when not thresholding.